rosdep/base.yaml

```diff
@@ -1350,6 +1355,11 @@ libexpat1-dev:
     portage:
       packages: [dev-libs/expat]
   ubuntu: [libexpat1-dev]
+libfcl-dev:
+  arch: [fcl]
+  debian: [libfcl-dev]
+  fedora: [fcl]
```

Comment:
Yeah, I don't see it here either: https://admin.fedoraproject.org/pkgdb/packages/?motif=fcl